docker-compose.yml

```diff
@@ -26,6 +26,13 @@ services:
     image: postgres:9.4-alpine
     volumes:
       - "./db/storage/:/var/lib/postgresql/data"
+  wraith:
```

Comment:
how does this work vs the above script? do we need one, the other, both?

Reply:
I am not the person to ask for docker information xD https://github.com/Marri/glowfic/commit/52902c890d69aff319c3e75aa7139537eb331a10#diff-0d347e7749477e6dc8fe8d71e84c020c tho